views/account/profile/index.ejs

```diff
@@ -14,33 +14,28 @@
     >
       <div class="row col-12">
         <main class="px-3">
-          <h1 class="display-4">Goon up Brah.</h1>
-          <h1>Hello World!</h1>
-          <p>This file was served with Express.</p>
-          <p>Here is your information:</p>
-          <ul>
-            <li>First Name: <%= user.first_name %></li>
-            <li>Last Name: <%= user.last_name %></li>
-            <li>Email: <%= user.email %></li>
-            <li>Password: <%= user.password %></li>
-            <li>
-              Settings:
-              <ul>
-                <% var settings = JSON.parse(user.settings);
-                Object.keys(settings).forEach( (key) => { %>
-                <li><%= key %>: <%= settings[key] %></li>
-                <% }); %>
-              </ul>
-            </li>
-          </ul>
-
-          <a
-            href="#"
-            class="btn btn-warning btn-lg"
-            role="button"
-            aria-disabled="true"
-            >Generate</a
-          >
+          <h1 class="display-4">Social Media Accounts</h1>
+          <h1 id="edit-existing-header">Edit Existing</h1>
+          <table id="accounts-table" class="table table-dark table-borderless">
+            <thead>
+                <tr>
+                    <th>Label</th>
+                    <th>Value</th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody id="accounts-table-body">
```

Comment:
I like how this looks a lot more. Instead of having the delete button, we can have a FA(Font awesome) icon of a trashcan instead to make it look cooler

Reply:
i totally agree with this idea. we do a similar approach to social media accounts like matching up the platform with its respective logo. for example "Instagram" can be replaced with the instagram logo. something to consider down the lane maybe?